concent-vm/configure.yml

```diff
@@ -196,3 +196,12 @@
             - storage
           become:      yes
           become_user: postgres
+
+        - name:   Copy helper scripts for managing Concent
+          command: cp                                                            \
+              --recursive                                                        \
+              --preserve                                                         \
```

Comment:
You can use `copy` command for copying files around inside maching. You just have to set `remote_src: yes`.

Reply:
OK, I see that the recursive copy is the problem. Let's leave is as is then